core/src/main/java/org/apache/spark/shuffle/sort/UnsafeShuffleWriter.java

```diff
@@ -163,6 +167,13 @@ public long getPeakMemoryUsedBytes() {
     return peakMemoryUsedBytes;
   }
 
+  public RowBasedChecksum[] getRowBasedChecksums() {
+    return rowBasedChecksums;
+  }
+  public long getAggregatedChecksumValue() {
```

Comment:
Nit: missing new line to separate those methods.

Reply:
updated